cpp/src/tests/renumber/renumber_test.cu

```diff
@@ -260,24 +266,39 @@ TEST_F(RenumberingTest, Random100KVertexSet)
   size_t min_id = unique_verts;
   size_t max_id = 0;
 
+  size_t cnt = 0;
   for (size_t i = 0 ; i < num_verts ; ++i) {
     min_id = min(min_id, tmp_results[i]);
     max_id = max(max_id, tmp_results[i]);
-    EXPECT_EQ(tmp_map[tmp_results[i]], src_data[i]);
+    if (tmp_map[tmp_results[i]] != src_data[i])
+      ++cnt;
+
+    if (cnt < 20)
+      EXPECT_EQ(tmp_map[tmp_results[i]], src_data[i]);
```

Comment:
I suspect this `if` was added for debug. It seems to me that when cnt is greater than 0 the data is corrupted, correct?

Reply:
I mean should we have a `EXPECT_EQ(0, cnt);`?

Reply:
Nevermind I think I get the logic it should capture the failure properly